src/plugins/navigation_embeddable/public/components/navigation_embeddable_component.scss

```diff
@@ -0,0 +1,58 @@
+.navEmbeddableComponent {
+  font-weight: $euiFontWeightRegular;
+
+  .navigationLink {
+    &.dashboardLinkError {
+      max-width: fit-content; // added this so that the error tooltip shows up **right beside** the link label
+
+      &.dashboardLinkError--noLabel .euiListGroupItem__button {
+        font-style: italic;
+      }
+
+      .dashboardLinkIcon {
+        margin-right: $euiSizeS;
+      }
+    }
+
+    &.navigationLinkCurrent {
+      border-radius: 0;
+      .euiListGroupItem__text {
+        cursor: default;
+        font-weight: $euiFontWeightSemiBold;
+        color: $euiColorPrimary;
+      }
+    }
+  }
+
+  .verticalLayoutWrapper {
+    .navigationLink {
+      .euiListGroupItem__text {
+        line-height: 1.2rem;
+        min-block-size: 1.2rem;
+        padding-block: $euiSizeXS;
+      }
+
+      &.navigationLinkCurrent {
+        box-shadow: $euiColorPrimary 2px 0 0 inset;
+      }
+    }
+  }
```

Comment:
![Jul-25-2023 09-28-41](https://github.com/elastic/kibana/assets/8698078/caaa0831-1cdf-4f89-8c6a-1f8e80ce9bc4)

Reply:
Should we add a tooltip on hover that shows the full label in case it's cut off?

The tooltip could even show the Dashboard title or description maybe.

Reply:
Perhaps, if the dashboard has a description, the tooltip shows the description - otherwise, if it doesn't have a description, we default back to the title. That way, the user always has some context provided about what dashboard is being pointed to even if the link has a custom label by the author and/or the title is cut off 🤔

Reply:
That's not a bad idea! I suppose we don't have any sort of advanced tooltip in EUI that could show all of that information?

Reply:
Ooo, I didn't know tooltips could have titles 😅 Yeah, that would definitely work! Let me try it out....

Reply:
How do we feel about this? 🤔 


https://github.com/elastic/kibana/assets/8698078/a1133043-1068-44e2-a64a-68cc46d89fd2

Unfortunately, because I can't use the `EuiListGroupItem` tooltip if we want to use the tooltip `title`, that means that the tooltip is **only** attached to the **text** of the button/link and not the button itself (i.e. it wraps the **text** of the button and not the button itself).... Not sure if we would want wait on this and make a request to EUI to allow adding a tooltip title directly through the `EuiListGroupItem` component instead?

Reply:
Looks nice!

It might not be a bad idea to file an issue with EUI for this, and if they do end up getting to it we can upgrade. In the meantime the tooltip on text only is _okay_ IMO.

Reply:
Filed an [issue](https://github.com/elastic/eui/issues/7000) with EUI :+1: